src/pf-modal/index.html

```diff
@@ -0,0 +1,57 @@
+<!doctype html>
+<html>
+<head>
+    <title>pf-tooltip example</title>
+    <link rel="stylesheet" href="/dist/css/patternfly.css">
+    <link rel="stylesheet" href="/dist/css/patternfly-additions.css">
+    <link rel="stylesheet" href="/dist/css/patternfly-webcomponents.css">
+    <script src="//cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/0.7.22/webcomponents.js"></script>
+    <script src="/dist/js/pf-utils.js"></script>
+    <script src="/dist/js/pf-modal.js"></script>
+</head>
+<body>
+<div class="container">
+    <div class="page-header">
+      <h1>Modals</h1>
+    </div>
+    <hr>
+    
+    <button class="btn btn-default" id="btn-toggle-modal">Launch demo modal</button>
+    <pf-modal targetSelector="#btn-toggle-modal" backdrop keyboard>
```

Comment:
I think "provide all this raw HTML themselves" is just what users want most from pf-modal because children custom elements(pf-modal-header, pf-modal-body, pf-modal-footer) provide users with the maximal free space to construct their own internal structure of pf-modal 😊